### PR TITLE
update version to 1.0.1

### PR DESCRIPTION
**Pull request by elharo**
@chanseokoh @briandealwis 

Do we need to update this anywhere else?

### CODE REVIEW

**Comment by chanseokoh** [comment]
I see a product version "1.0.0" in `gcp-repo/product`, but I'm not sure what that means or how it relates to a feature version.

I also see a category version "1.0.0" in `gcp-repo/metadata.p2.inf`, but I get the feeling that a category is a different kind that may keep its own version. Not sure though.

**Comment by briandealwis** [comment]
The category in `gcp-repo/metadata.p2.inf` may be worth changing since it's reflected in the installation UI:

screen shot 2017-03-09 at 4 35 55 pm

The `gcp-repo/metadata.product` can be left untouched: it won't be visible except to those who know how to use p2 tools.

**Comment by chanseokoh** [comment]
> The gcp-repo/metadata.product can be left untouched: it won't be visible except to those who know how to use p2 tools.

Could be useful to put a comment inside `gcp-repo/metadata.product`.

**Comment by elharo** [comment]
PTAL. Updated both.

File: gcp-repo/metadata.p2.inf [hunk 1]
```diff
@@ -27,7 +27,7 @@ units.1.properties.5.name=org.eclipse.equinox.p2.doc.url
 units.1.properties.5.value=https://cloud.google.com/eclipse/docs/
 units.1.provides.1.namespace=org.eclipse.equinox.p2.iu
 units.1.provides.1.name=com.google.cloud.tools.eclipse.category
-units.1.provides.1.version=1.0.0
+units.1.provides.1.version=1.0.1
```
Comment (briandealwis) [inline gcp-repo/metadata.p2.inf]: I think you need to change the `requires.1.range` up above.  This .p2.inf format hurts to read.  Basically this file associates p2 metadata to the `com.google.cloud.tools.eclipse.dist` product defined in the `gcp-repo/metadata.product` file.  
  - the initial `requires.1` adds a requirement on the product IU for an IU with certain capabilities; think of the _namespace_ as being a the capability _type_, and the _name_ is the capability identifier.  So this `requires.1` says the product requires our new category
  - the `units.N` defines new IUs; the `units.N.provides.M` describes the capabilities of this IU. In this case this IU provides a category definition.  The category values are in the properties.
  - the category IU also has some requirements to pull in a feature

Comment (chanseokoh) [inline gcp-repo/metadata.p2.inf]: @briandealwis there is also `units.1.version=1.0.0` above. That is left untouched?

Comment (briandealwis) [inline gcp-repo/metadata.p2.inf]: I don't know! I think we have to try it and maybe change the unimportant versions to 999.999.999 to indicate that they don't matter.

**Comment by elharo** [comment]
PTAL

File: gcp-repo/metadata.p2.inf (hunk 1 above)
Comment (briandealwis) [inline gcp-repo/metadata.p2.inf]: The `units.1.version` determines what's shown the user.
```diff
diff --git a/gcp-repo/metadata.p2.inf b/gcp-repo/metadata.p2.inf
index 010b3c0..c234b59 100644
--- a/gcp-repo/metadata.p2.inf
+++ b/gcp-repo/metadata.p2.inf
@@ -14,7 +14,7 @@ requires.1.range=[1.0.0,1.0.1]
 requires.1.greedy=true
 
 units.1.id=com.google.cloud.tools.eclipse.category
-units.1.version=1.0.1
+units.1.version=999
 units.1.properties.1.name=org.eclipse.equinox.p2.type.category
 units.1.properties.1.value=true
 units.1.properties.2.name=org.eclipse.equinox.p2.name
```
produces:
screen shot 2017-03-09 at 11 12 48 pm

**Pull request merged**